app/components/contents_component.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
 class ContentsComponent < ApplicationComponent
-  def initialize(cocina:, document:)
+  def initialize(cocina:, document:, state_service:)
```

Comment:
Since the presenter holds `cocina`, `document` and `state_service`, how about changing this to accept one argument, the presenter?

Reply:
done